frontend/sac-mobile/components/Wordmark.tsx

```diff
@@ -0,0 +1,45 @@
+import { Pressable, StyleSheet, Text, View } from 'react-native';
+
+import { Button } from '@/components/button';
+
+type WordmarkProps = {
+    textColor?: string;
+    backgroundColor?: string;
+    button?: boolean;
+    func?: () => void;
+    title?: string;
+};
+
+const Wordmark = (props: WordmarkProps) => {
```

Comment:
destructure the props rather than just leaving it as a props object saves us from doing props.xyz